### ISSUE TITLE
Create preview images for the cassini_uvis_solarocc_beckerjarmak2023 bundle

### ISSUE THREAD

**Issue by rfrenchseti** [SETI/rms-data-projects#117]
Create preview images (wiggly line plots) for the `cassini_uvis_solarocc_beckerjarmak2023` PDS4 bundle so that we can incorporate it into OPUS.

**Comment by matthewtiscareno** [comment]
Isn't this one essential for incorporating the data set into OPUS while #116 is optional?  Shouldn't that be reflected in different priority tags?

**Comment by rfrenchseti** [comment]
Yes, feel free to prioritize as you see fit. Since priorities are global, I didn't want to assume one project was less/more important than other things people were doing.

**Comment by mseritan** [comment]
I'd like feedback on the structure of the titles for the full-size previews. They currently read, for example:

> Cassini UVIS EUV Solar Occultation
> 57-117 nm, Saturn's Rings, Ingress

The other information available from each file is the date on which it was taken, in year/number-day-of-year format (ex. 2005_159 means 8 June 2005). Should this be included in the titles as well, and in which format? A couple examples are:

> Cassini UVIS EUV Solar Occultation, 2005, 159th day
> 57-117 nm, Saturn's Rings, Ingress

or:

> Cassini UVIS EUV Solar Occultation, 8 Jun. 2005
> 57-117 nm, Saturn's Rings, Ingress

**Comment by mseritan** [comment]
For the time being, I've created the titles to read (for example):
> Cassini UVIS EUV Solar Occultation, 2005-159
> 57-117 nm, Saturn's Rings, Ingress

**Comment by mseritan** [comment]
These previews have been re-generated. They now have the correct output directory structure (mirrors the bundle's directory structure), and the x-axes have been redone so that they use a column in the data table ("Ring Radius Mid Integration") rather than generated values between two end points that were taken from the table labels.

**Comment by mseritan** [comment]
The previews have been put into a new `previews` directory on `pdsdata-staging`.

I've structured everything inside `previews/cassini_uvis_solarocc_beckerjarmak2023` to mirror the bundle itself: 41 sets of four images are directly inside `cassini_uvis_solarocc_beckerjarmak2023`, and 1 set of four images inside a subdirectory called `supplemental`.

**Comment by rfrenchseti** [comment]
Are the Y units really "counts/sec" instead of just "counts"?

To agree with the format of the Uranus previews directory, all previews should be under `data` to mirror the structure of the bundle.

The proper suffix for the medium previews is `_med`, not `_medium`.

A huge number of `._` files were copied into the previews directory. These are Mac internal files and should be excluded from rsync. I thought we had set up admin so that these files wouldn't be created on a remote NFS filesystem. Did that setting get changed, or maybe it never worked? Either way every rsync command ever done from a Mac should explicitly exclude these types of files.

**Comment by mseritan** [comment]
As of last week, this task has almost been completed. The previews are on `pdsdata-staging` awaiting review by @rfrenchseti. Once approved, I will rsync them to `pdsdata-production`.

**Comment by rfrenchseti** [comment]
Approved.
